views/backend/settings.php

```diff
@@ -35,18 +34,25 @@
             </tbody>
         </table>
 
-
     </div>
     <div class="col m6 s12">
         <h4>Ajouter un modo</h4>
 
         <?php
             if(isset($_POST['submit'])){
-
-                $name = htmlspecialchars(trim($_POST['name']));
-                $email = htmlspecialchars(trim($_POST['email']));
-                $email_again = htmlspecialchars(trim($_POST['email_again']));
-                $role = htmlspecialchars(trim($_POST['role']));
+                if(isset($_POST['name'])){
+                    $name = htmlspecialchars(trim($_POST['name']));
+                }
+                if(isset($_POST['email'])){
+                    $email = htmlspecialchars(trim($_POST['email']));
+                }
+                if(isset($_POST['email_again'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)